buildenv/jenkins/Jenkinsfile_personal

```diff
@@ -72,9 +72,7 @@ pipeline {
                 	echo 'Building tests...'
                 	script {
 	                	if (JAVA_VERSION == 'SE80') {
-							sh 'chmod 755 ${JAVA_BIN}/java'
-							sh 'chmod 755 ${JAVA_BIN}/../../bin/javac'
-							sh 'chmod 755 ${JAVA_BIN}/../../bin/java'
+							sh 'chmod -R 755 $WORKSPACE/openjdkbinary/j2sdk-image/'
```

Comment:
Is it possible to have this chmod in get.sh, so we do not need to do it in two separate files. By putting it in get.sh, it will solve the problem during local run as well.

Reply:
It should be possible. I'd have to test it with a build.

Reply:
I've moved the chmod to get.sh. Tested with z/OS and it works fine. Thanks for the suggestion.